packages/atomic/src/components/search/facets/atomic-facet/e2e/atomic-facet.e2e.ts

```diff
@@ -5,26 +5,73 @@ test.describe('when clicking facet search "More matches for"', () => {
     await facet.load({story: 'low-facet-values'});
   });
   test('should display an increasing number of matches', async ({facet}) => {
-    await facet.getFacetSearch.click();
-    await facet.getFacetSearch.pressSequentially('p');
+    await facet.facetSearch.click();
+    await facet.facetSearch.pressSequentially('p');
     await expect
       .poll(async () => {
-        return await facet.getFacetValue.count();
+        return await facet.facetValue.count();
       })
       .toBeGreaterThanOrEqual(2);
 
     await facet.facetSearchMoreMatchesFor.click();
     await expect
       .poll(async () => {
-        return await facet.getFacetValue.count();
+        return await facet.facetValue.count();
       })
       .toBeGreaterThanOrEqual(4);
 
     await facet.facetSearchMoreMatchesFor.click();
     await expect
       .poll(async () => {
-        return await facet.getFacetValue.count();
+        return await facet.facetValue.count();
       })
       .toBeGreaterThanOrEqual(6);
   });
 });
+
+test.describe('when selecting the "Show more" button', () => {
```

Comment:
While the tests themselves are fine, I feel like they're oddly organized. Also, the feature coverage seems incomplete to me.

I think we should have something like this:

- when the "Show more" button has not been selected
  - should be accessible
  - should not display the "Show less" button
  - when additional facet values are available
    - should display the "Show more" button
    - should have the configured number of facet values
  - when no additional facet values are available
    - should not display the "Show more" button
    - should have as many or fewer than the configured number of facet values
- when the "Show more" button has been selected
  - should be accessible
  - should have facet values sorted alphabetically
  - when additional facet values are available
    - should display the "Show more" button
    - should have the configured number of facet values * (1 + the number of times the button was selected)
  - when no additional facet values are available
    - should have the total number of available facet values
    - should not display the "Show more" button
  - should display the "Show less" button
  - when the "Show less" button is selected
    - should not display the "Show less" button
    - should display the "Show more" button
    - should have the configured number of facet values

Reply:
remade the test suite to fix exactly that description, thanks for the help !